master.css

```diff
@@ -350,7 +365,18 @@ label[for="sort"] {
     width: 260px;
 }
 .num-features {
-    position: absolute; top: -15px; right: 0px; font-size: 11px;
+    position: absolute;
+    top: -15px;
+    right: 0px;
+    font-size: 11px;
+    left: 0px;
+    background-color: #eee;
+    border-bottom: 1px solid #fff;
```

Comment:
I was puzzled by the `border-bottom` line - specifically, that it isn't overridden in the ES6 table, leaving misaligned white lines beneath the percentages (in Firefox, at least).